src/southbridge/amd/sb700/sm.c

```diff
@@ -475,11 +482,13 @@ static void sb700_sm_read_resources(struct device *dev)
 	 * write address in set_resources() below.
 	 */
 	res = new_resource(dev, SB_MMIO_CFG_REG);
+	res->base  = SB_MMIO_BASE_ADDRESS;
 	res->size = 0x1000;
 	res->limit = 0xffffffffUL;
 	res->align = 12;
 	res->gran = 12;
-	res->flags = IORESOURCE_MEM | IORESOURCE_ASSIGNED;
+	res->flags = IORESOURCE_MEM | IORESOURCE_FIXED | IORESOURCE_ASSIGNED |
```

Comment:
Since this is fixed and stored, "Enable SB MMIO" in function below (`sb700_sm_set_resources`) is no longer necessary, please remove it.

I'm still not convinced on hardcoding the address, but if this is the case across all AMD boards, it is better to not make an exception here.

Reply:
Isn't it necessary to just inform the resource allocator about the resource presence anyways? So that it will not allocate any other resource here,

Reply:
Enabling SB MMIO removed from sb700_sm_set_resources

Reply:
@krystian-hebel can we resolve this conversation and close the PR?